### DISCUSSION TITLE
Persistent Connection?

### DISCUSSION THREAD

**Discussion by jeffery30162** [uNetworking/uWebSockets.js#656] [Q&A]
Are persistent connections supported?

**Comment by e3dio** [comment]
Websocket by definition is a persistent connection, this library goes a step further and automatically sends pings to clients at a set idle timeout to keep connections open

**Reply by e3dio** [reply]
You are probably referring to http, yes it does persistent connection

**Comment by uasan** [comment]
If the client or proxy uses the HTTP 1.0 protocol then a separate client connection will be created for each HTTP request.
